src/Passes/qir/qsharp/analysis.py

```diff
@@ -0,0 +1,52 @@
+import os
```

Comment:
Is this file documented in a readme somewhere?

Reply:
No, this is just a copy from the examples repo and it is WiP